lessons/AI/ai-statistical-lang-models/langs/en-us/pages/constructing-model.adoc

```diff
@@ -0,0 +1,88 @@
+= Constructing a Language Model
+
+Use @handout{old-lady-lyrics.adoc} to answer the questions, below. *Note that the title of the song is a part of the corpus!*
+
+== Did she swallow a fly?
+
+@n Let's say we want to know the likelihood that the old lady "swallowed a fly". We need a ratio!
+
+@vspace{1ex}
+
+
+@math{p(fly | swallowed a) =}
+@math{\frac
+	{\textit{count(swallowed a fly)}}
+	{\textit{count(swallowed a...)}}
+= {\frac{}{10}}}
+
+
+@n The likelihood that the old lady "swallowed a fly" in the excerpt is @fitb{5em}{@ifsoln{60%}}.
+
+== What about the other animals?
+
+@vspace{1ex}
+
+@n
+@math{p(spider | swallowed a) =}
+@math{\frac
+	{\textit{count(swallowed a spider)}}
+	{\textit{count(swallowed a...)}}
+= {\frac{}{10}}}
+
+@n
+@math{p(bird | swallowed a) =}
+@math{\frac
+	{\textit{count(swallowed a bird)}}
+	{\textit{count(swallowed a...)}}
+= {\frac{}{10}}}
+
+@n
+@math{p(cat | swallowed a) =}
+@math{\frac
+	{\textit{count(swallowed a cat)}}
+	{\textit{count(swallowed a...)}}
+= {\frac{}{10}}}
+
+
+@n Across the entire poem, the lady _most likely_ swallowed a @fitb{10em}{@ifsoln{fly (6/10)}}. The second most likely option is that she swallowed a @fitb{10em}{@ifsoln{cat (2/10)}}. There is an equal probability that she swallowed a @fitb{10em}{@ifsoln{spider (1/10)}} or a @fitb{10em}{@ifsoln{bird (1/10)}}.
+
+
+== To catch the... ?
+
+@vspace{1ex}
+
+@n
+@math{p(fly | to catch the) =}
+@math{\frac
+	{\textit{count(to catch the fly)}}
+	{\textit{count(to catch the...)}}
+= {\frac{}{6}}}
+
+@vspace{1ex}
+
+@n
+@math{p(spider | to catch the) =}
+@math{\frac
+	{\textit{count(to catch the spider)}}
+	{\textit{count(to catch the...)}}
+= {\frac{}{6}}}
+
+@vspace{1ex}
+
+@n
+@math{p(bird | to catch the) =}
+@math{\frac
+	{\textit{count(to catch the bird)}}
+	{\textit{count(to catch the...)}}
+= {\frac{}{6}}}
+
+@vspace{1ex}
+
+The lady was most likely trying to catch the @fitb{10em}{@ifsoln{fly (3/6)}}. It is the least likely that she was trying to catch the @fitb{10em}{@ifsoln{bird (1/6)}}.
+
+== Try it on your own and reflect
```

Comment:
I'm a little unclear why you're breaking with the format here and why you titled this section "reflect"